plugins/available/systemd.plugin.bash

```diff
@@ -0,0 +1,50 @@
+# Author: Arthur Miller
+# Created: 2017-08-08
+# License: Public Domain
+
+cite about-plugin
+about-plugin 'SystemD helper functions'
+
+# Reloads a systemwide service. If no agruments are given, reloads all services.
+# Requires admin priviledge.
+scrs(){
```

Comment:
A couple of general comments for all of these functions:

1. The name should be descriptive, don't use an abbreviation here. Use something like `systemd_reload_service`. If you want a shorter version of that, add a shorter alias in the `systemd.aliases.bash` after that.
1. Please add the meta tags for each function, these are used for describing the available functions. Please see https://github.com/Bash-it/bash-it/blob/master/plugins/available/base.plugin.bash#L21 for an example of what to add for each function.